src/app/machines/views/MachineList/MachineList.test.tsx

```diff
@@ -374,6 +375,7 @@ describe("MachineList", () => {
       </Provider>
     );
     expect(wrapper.find("Notification").exists()).toBe(true);
+    // eslint-disable-next-line testing-library/no-node-access
```

Comment:
This whole file needs to be migrated to RTL, but in the meantime I've disabled these warnings.